dbump_pgx/pgx.go

```diff
@@ -66,6 +66,8 @@ SET created_at = NOW();`
 
 // Exec is a method for Migrator interface.
 func (pg *Migrator) Exec(ctx context.Context, query string, args ...interface{}) error {
-	_, err := pg.conn.Exec(ctx, query)
-	return err
+	return pg.conn.BeginTxFunc(ctx, pgx.TxOptions{}, func(tx pgx.Tx) error {
+		_, err := tx.Exec(ctx, query, args...)
+		return err
+	})
```

Comment:
Let's keep this change separately. There must be a new feature to use tx/not for each migration step.

Reply:
Okay. Rolled back.